pvlib/tests/test_irradiance.py

```diff
@@ -1095,6 +1095,29 @@ def test_dirindex(times):
         equal_nan=True)
 
 
+@fail_on_pvlib_version("0.13")
+def test_dirindex_ghi_clearsky_deprecation():
+    with pytest.warns(pvlibDeprecationWarning, match='ghi_clear'):
+        times = pd.DatetimeIndex(['2014-06-24T18-1200'])
+        ghi = pd.Series([1038.62], index=times)
+        ghi_clearsky = pd.Series(
+            np.array([1042.48031487]),
+            index=times
+        )
+        dni_clearsky = pd.Series(
+            np.array([939.95469881]),
+            index=times
+        )
+        zenith = pd.Series(
+            np.array([10.56413562]),
+            index=times
+        )
+        pressure, tdew = 93193, 10
+        irradiance.dirindex(
+            ghi=ghi, ghi_clearsky=ghi_clearsky, dni_clearsky=dni_clearsky,
```

Comment:
```suggestion
            ghi=ghi, ghi_clearsky=ghi_clearsky, dni_clear=dni_clearsky,
```
Nitpicking about the warnings. Not problematic as of now, both warnings appear and the `ghi_clear` is matched, but since there already is a dedicated test for deprecated `dni_clearsky` (`test_dirindex_dni_clearsky_deprecation`, Line 1153) it is more atomic to limit the one the test is for.